File: ncDados/dTipoNotaFiscal.vb
Namespace nsDados

  Public Class ColecaoTipoNotaFiscal
    Inherits List(Of dTipoNotaFiscal)
  End Class

  Public Class dTipoNotaFiscal

    Private _cid As Integer
    Private _codigo As String
    Private _descricao As String
    Private _modelo As String
    Private _modeloDescricao As String

    Public Property cid() As Integer
      Get
        Return _cid
      End Get
      Set(ByVal value As Integer)
        _cid = value
      End Set
    End Property

    Public Property codigo() As String
      Get
        Return _codigo
      End Get
      Set(ByVal value As String)
        _codigo = value
      End Set
    End Property

    Public Property descricao() As String
      Get
        Return _descricao
      End Get
      Set(ByVal value As String)
        _descricao = value
      End Set
    End Property

    Public Property modelo() As String
      Get
        Return _modelo
      End Get
      Set(ByVal value As String)
        _modelo = value
      End Set
    End Property

    Public Property modeloDescricao() As String
      Get
        Return _modeloDescricao
      End Get
      Set(ByVal value As String)
        _modeloDescricao = value
      End Set
    End Property

  End Class

End Namespace

File: Modelos/dTipoNotaFiscal.cs
using System;
using System.Collections.Generic;

namespace Modelos
{
    public class ColecaoTipoNotaFiscal : List<dTipoNotaFiscal>
    {
    }

    public class dTipoNotaFiscal
    {
        private int _cid;
        private string _codigo;
        private string _descricao;
        private string _modelo;
        private string _modeloDescricao;

        public int cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string codigo
        {
            get => _codigo;
            set => _codigo = value;
        }

        public string descricao
        {
            get => _descricao;
            set => _descricao = value;
        }

        public string modelo
        {
            get => _modelo;
            set => _modelo = value;
        }

        public string modeloDescricao
        {
            get => _modeloDescricao;
            set => _modeloDescricao = value;
        }
    }

}
